### DISCUSSION TITLE
Getting a white screen and Server Errors after upgrading

### DISCUSSION THREAD

**Discussion by MarkBurvs** [directus/directus#5211] [Archive]
I'm trying to upgrade Directus from 9.0.0-rc.44 to 9.0.0-rc.60 but following the upgrade, I just get a blank white screen with several errors in the console all saying "Failed to load resource: the server responded with a status of 500 (Internal Server Error)".

Any suggestions about how I can diagnose what's going wrong?

**Comment by scouech** [comment]
@MarkBurvs have you restarted/reloaded node / pm2 ?

**Reply by MarkBurvs** [reply]
Yep. I restarted pm2, then rebooted the whole server and neither worked. I then restored the droplet from the last backup and tried the update a second time making sure to shut down the pm2 service first, but same result.

**Reply by scouech** [reply]
and on the pm2 list, what does it say? Is it using any memory at all? 
I had the same problem yesterday when I upgraded - removing the service from pm2 and re-adding it fixed the issue

**Reply by MarkBurvs** [reply]
Yeah pm2 said it was running and using comparable level of memory to before I ran the upgrade. I will try removing and re-adding it though.

**Reply by MarkBurvs** [reply]
nope that didn't work. Still getting the same.

![Screenshot 2021-04-23 at 11 30 15](https://user-images.githubusercontent.com/9461289/115863852-f7095680-a42d-11eb-81f1-01a6f82802d5.png)

**Reply by rijkvanzanten** [reply]
@MarkBurvs Did you migrate the database using `directus database migrate:latest` or `directus bootstrap`?

**Reply by MarkBurvs** [reply]
@rijkvanzanten Ah no I didn't! Didn't know that was a thing that needed doing.

**Reply by MarkBurvs** [reply]
Yep that was it!
